frontend/fontlist.lua

```diff
@@ -48,10 +50,28 @@ local kindle_fonts_blacklist = {
     ["TBMinchoBold_213.ttf"] = true,
     ["TBMinchoMedium_213.ttf"] = true,
     ["STKaiMedium.ttf"] = true,
+    ["Amazon-Ember-Bold.ttf"] = false,
+    ["Amazon-Ember-BoldItalic.ttf"] = false,
+    ["Amazon-Ember-Heavy.ttf"] = true,
+    ["Amazon-Ember-HeavyItalic.ttf"] = true,
+    ["Amazon-Ember-Medium.ttf"] = true,
+    ["Amazon-Ember-MediumItalic.ttf"] = true,
+    ["Amazon-Ember-Regular.ttf"] = false,
+    ["Amazon-Ember-RegularItalic.ttf"] = false,
+    ["AmazonEmberBold-Bold.ttf"] = true,
+    ["AmazonEmberBold-BoldItalic.ttf"] = true,
+    ["AmazonEmberBold-Italic.ttf"] = true,
+    ["AmazonEmberBold-Regular.ttf"] = true,
+    ["Caecilia_LT_65_Medium.ttf"] = false,
+    ["Caecilia_LT_66_Medium_Italic.ttf"] = false,
+    ["Caecilia_LT_75_Bold.ttf"] = false,
+    ["Caecilia_LT_76_Bold_Italic.ttf"] = false,
     ["Caecilia_LT_67_Cond_Medium.ttf"] = true,
     ["Caecilia_LT_68_Cond_Medium_Italic.ttf"] = true,
     ["Caecilia_LT_77_Cond_Bold.ttf"] = true,
     ["Caecilia_LT_78_Cond_Bold_Italic.ttf"] = true,
+    ["Futura-Bold.ttf"] = true,
```

Comment:
Must be annoying for Kindle users that they use normal font names for these weird fonts.